pkg/kn/commands/domain/create.go

```diff
@@ -33,7 +33,7 @@ func NewDomainMappingCreateCommand(p *commands.KnParams) *cobra.Command {
 		Use:   "create NAME",
 		Short: "Create a domain mapping",
 		Example: `
-  # Create a domain mappings 'hello.example.com' for Knative service 'hello' with TLS secret set
+  # Create a domain mappings 'hello.example.com' for Knative service 'hello' with TLS secret set (Beta)
```

Comment:
I wonder whether we should mark the domain management as "Beta" or whether we say that it's already GA ? I think we can move it to GA level as the UI is straightforward and not really contentious.

Reply:
+1 to GA